angular/core/models/group_model.coffee

```diff
@@ -169,3 +178,10 @@ angular.module('loomioApp').factory 'GroupModel', (DraftableModel, AppConfig) ->
 
     isSubgroupOfSecretParent: ->
       @isSubgroup() && @parent().privacyIsSecret()
+
+    setupComplete: ->
```

Comment:
Happy to make that change. Tests don't seem to be working unfortunately - would you mind walking me through that?